exposed/src/main/kotlin/org/jetbrains/exposed/sql/transactions/TransactionApi.kt

```diff
@@ -68,13 +70,13 @@ interface TransactionManager {
             }
         }
 
-        internal fun managerFor(database: Database) = registeredDatabases[database]
+        internal fun managerFor(database: Database?) = if (database != null) registeredDatabases[database] else manager
```

Comment:
Why so if you expect to fall with exception later in `Database?.transactionManager`?